src/Console/Command/CheckCommand.php

```diff
@@ -40,6 +41,7 @@ protected function configure()
                     ),
                     new InputOption('no-cache', null, InputOption::VALUE_NONE, 'Disable rule set cache'),
                     new InputOption('cache-dir', null, InputOption::VALUE_REQUIRED, 'Cache directory', '.rules/'),
+                    new InputOption('log-html', null, InputOption::VALUE_OPTIONAL, 'Log output in HTML format to file.'),
```

Comment:
I agree. I will change this. The intention was to not force the user to enter something. But obviously   it is unnecessary. Thanks for the input.